daemon/cluster/cluster.go

```diff
@@ -629,13 +626,25 @@ func (c *Cluster) isActiveManager() bool {
 	return c.conn != nil
 }
 
+// errNoManager returns error describing why manager commands can't be used.
+// Call with read lock.
+func (c *Cluster) errNoManager() error {
+	if c.node == nil {
+		return fmt.Errorf("This node is not participating as a Swarm manager. Use \"docker swarm init\" or \"docker swarm join --manager\" to connect this node to Swarm and try again.")
+	}
+	if c.node.Manager() != nil {
+		return fmt.Errorf("This node is not participating as a Swarm manager. No active connection to the control API.")
```

Comment:
"Manager is being prepared or has trouble connecting to the cluster." ? Yes, this isn't really actionable(unless you want to leave the cluster) but thats why it's important to separate it from the cases where user just forgot to type a command.